tests/test_astrometryModel.py

```diff
@@ -0,0 +1,352 @@
+
+            src = dataRef.get("src", flags=lsst.afw.table.SOURCE_IO_NO_FOOTPRINTS, immediate=True)
+            goodSrc = sourceSelector.selectSources(src)
+            # Need memory contiguity to do vector-like things on the sourceCat.
+            goodSrc = goodSrc.sourceCat.copy(deep=True)
+
+            visitInfo = dataRef.get('calexp_visitInfo')
+            detector = dataRef.get('calexp_detector')
+            ccdId = detector.getId()
+            wcs = dataRef.get('calexp_wcs')
+            bbox = dataRef.get('calexp_bbox')
+            filt = dataRef.get('calexp_filter')
+            filterName = filt.getName()
+            photoCalib = lsst.afw.image.PhotoCalib(100.0, 1.0)
+
+            self.catalogs.append(goodSrc)
+            self.associations.addImage(goodSrc,
+                                       wcs,
+                                       visitInfo,
+                                       bbox,
+                                       filterName,
+                                       photoCalib,
+                                       detector,
+                                       visit,
+                                       ccdId,
+                                       jointcalControl)
+
+        # Have to set the common tangent point so projectionHandler can use skyToCTP.
+        centers = [ccdImage.getBoresightRaDec() for ccdImage in self.associations.getCcdImageList()]
+        # TODO DM-4404: this gets cleaner once jointcal.point is the same as afw.point
+        commonTangentPoint = lsst.afw.geom.averageSpherePoint(centers)
+        self.associations.setCommonTangentPoint(commonTangentPoint.getPosition(lsst.afw.geom.degrees))
+
+        self.projectionHandler = lsst.jointcal.OneTPPerVisitHandler(self.associations.getCcdImageList())
+
+        self.associations.associateCatalogs(matchCut)
+        self.associations.prepareFittedStars(minMeasurements)
+        self.associations.deprojectFittedStars()
+
+    def _prepModels(self):
+        """Call this after model1 and model2 are created, to call assignIndices,
+        and instantiate the fitters.
+        """
+        posError = 0.02  # in pixels
+        # have to call this once or offsetParams will fail because the transfo indices aren't defined
+        self.model1.assignIndices("Distortions", self.firstIndex)
+        self.fitter1 = lsst.jointcal.AstrometryFit(self.associations, self.model1, posError)
+
+        # have to call this once or offsetParams will fail because the transfo indices aren't defined
+        self.model2.assignIndices("Distortions", self.firstIndex)
+        self.fitter2 = lsst.jointcal.AstrometryFit(self.associations, self.model2, posError)
+
+    def testMakeSkyWcsModel1(self):
+        self._testMakeSkyWcs(self.model1, self.fitter1, self.inverseMaxDiff1)
+
+    def testMakeSkyWcsModel2(self):
+        self._testMakeSkyWcs(self.model2, self.fitter2, self.inverseMaxDiff2)
+
+    def _testMakeSkyWcs(self, model, fitter, inverseMaxDiff):
```

Comment:
Looks like this is a test that has been disabled by the leading underscore. It should be renamed.